tool.go

```diff
@@ -771,7 +771,7 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 		allArgs = []string{"--require", "source-map-support/register"}
 		if err := exec.Command("node", "--require", "source-map-support/register", "--eval", "").Run(); err != nil {
 			if !quiet {
-				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Use Node.js 4.x with source-map-support module for nice stack traces.")
+				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Install source-map-support module for nice stack traces. See https://github.com/gopherjs/gopherjs#installation-and-usage.")
```

Comment:
> Is Node 4.x a minimum requirement for that module? If so, it's probably worth still mentioning that, but possibly not in the error message itself.

It is still mentioned at the linked page. See https://github.com/gopherjs/gopherjs#installation-and-usage:

> If you want to use `gopherjs run` or `gopherjs test` to run the generated code locally, install Node.js 4.x and the module `source-map-support`:
>
> ```
> npm install --global source-map-support
> ```

> I wonder if it would also be better to link directly to the syscalls page (https://github.com/gopherjs/gopherjs/blob/master/doc/syscalls.md)?

But that page doesn't talk about `source-map-support` module at all. Pointing to it would be very confusing.

> Or make it more prominent on the page linked to (possibly with an anchor tag for direct linking)?

The Installation and Usage section is only 4 paragraphs long. Does it really need to be broken down into smaller subsections, or can we expect people to read the whole thing?